apps/web/src/screens/HomePage/index.tsx

```diff
@@ -84,10 +88,13 @@ const HomePage: React.FC<{}> = () => {
           toast(
             <Toast
               iconName={"alert-circle-outline"}
-              message={"Success! Redemption code applied to your account. Please refresh to see changes."}
+              message={
+                "Success! Redemption code applied to your account. Please refresh to see changes."
+              }
               fill={"green"}
             />
           );
+          props.history.push("/");
```

Comment:
There may be other query parameters that we don't want to squash

Reply:
Disagree, if they are doing a redemption code that should their only parameter.

Reply:
Not necessarily. For example, auth0 uses the `code` query parameter for login which will squash. The argument can be made that they will be logged in before running this, but future libraries can use this behavior for other things and we will then transparently fail

Reply:
On further review, I agree with you, though I am not sure we will ever see this problem, authentication is handled earlier than redemption code stuff. I will fix this